service/ws/utils.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2022-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package ws
+
+import (
+	"bytes"
+	"encoding/base32"
+
+	"github.com/pborman/uuid"
+)
+
+const charset = "ybndrfg8ejkmcpqxot1uwisza345h769"
+
+var encoding = base32.NewEncoding(charset)
+
+// NewID is a globally unique identifier.  It is a [A-Z0-9] string 26
+// characters long.  It is a UUID version 4 Guid that is zbased32 encoded
+// with the padding stripped off.
+func newID() string {
```

Comment:
any reason we're not using `model.NewId()`?

Reply:
The reason is I am a firm believer of [A little copying is better than a little dependency.](https://go-proverbs.github.io/)
We don't want to couple the service with server's `model` if we can avoid it cause they don't share anything.